src/main/resources/templates/statistics/statisticsHome.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="cs" xmlns:th="http://www.thymeleaf.org">
+<head th:replace="fragments/head :: head('Statistiky')">
+</head>
+<body>
+<div th:insert="fragments/organizerHeader :: header(statistics)"></div>
+<div class="container">
```

Comment:
![image](https://user-images.githubusercontent.com/43091754/61204542-b70c3000-a6ed-11e9-85ea-393e77efbf95.png)
Tady by to chtělo trochu odsadit od toho headeru možná :cake:

Reply:
FE predelan